config/ImathSetup.cmake

```diff
@@ -9,6 +9,8 @@ if(NOT "${CMAKE_PROJECT_NAME}" STREQUAL "${PROJECT_NAME}")
   message(STATUS "Imath is configuring as a cmake sub project")
 endif()
 
+option(IMATH_USE_DEFAULT_VISIBILITY "Makes the compile use default visibility (by default compiles tidy, hidden-by-default)"     OFF)
```

Comment:
The fact that the default visibility is not default by default makes my brain hurt :) Might be time for a wee dram of scotch.

Especially when the bit below says if (NOT IMATH_USE_DEFAULT_VISIBILITY), and I'm puzzling over I need to turn off the default to get non default default

LOL, sorry

Reply:
PS, this approval is my thumbs up pending resolving Cary's comments